Blahgger/Blahgger/Data/BlahggerData.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Blahgger.Models;
+using System.Data.SqlClient;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Web.Security;
+
+namespace Blahgger.Data
+{
+    /// <summary>
+    /// This class contains methods for binding data from the database to models.
+    /// </summary>
+    public class BlahggerData
+    {
+        private static BlahggerData singleton = null;
+
+        private BlahggerData() { }
+
+        public static BlahggerData GetInstance()
+        {
+            if (singleton == null)
+            {
+                singleton = new BlahggerData();
+            }
+            return singleton;
+        }
```

Comment:
Sorry about the singleton data class. This was before our conversation about stateless programming.

Reply:
No worries at all! Thanks for the reminder comment :)